src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -1090,4 +1097,73 @@ public function setEchoFormat($format)
     {
         $this->echoFormat = $format;
     }
+
+    /**
+     * Compile the switch statements into valid PHP.
+     * @param  string  $variable
+     *
+     * @return string
+     */
+    public function compileSwitch($variable)
+    {
+        return "<?php switch({$variable}){ ";
```

Comment:
checkout the warning in the doc, may be its well explained there
http://www.php.net/manual/en/control-structures.alternative-syntax.php